src/custom-types.json

```diff
@@ -75,7 +75,7 @@
   "application/javascript": {
     "charset": "UTF-8",
     "compressible": true,
-    "extensions": ["mjs"],
+    "extensions": ["mjs","ejs"],
```

Comment:
Thank you! From https://github.com/jshttp/mime-db#contributing , please add a link in the `sources` here of a primary source that notes that the `.ejs` file extension is the `application/javacript` mime type (just like there is already a source link here for `.mjs`).

Reply:
Not to sure about a definitive source, I just have found it used in several articles relating to Babel et.al. I am going to move to .mjs !